core/server/controllers/admin.js

```diff
@@ -32,27 +30,7 @@ adminControllers = {
             });
         }
 
-        updateCheck().then(function then() {
-            return updateCheck.showUpdateNotification();
-        }).then(function then(updateVersion) {
-            if (!updateVersion) {
-                return;
-            }
-
-            var notification = {
-                type: 'upgrade',
-                location: 'settings-about-upgrade',
-                dismissible: false,
-                status: 'alert',
-                message: i18n.t('notices.controllers.newVersionAvailable',
-                                {version: updateVersion, link: '<a href="http://support.ghost.org/how-to-upgrade/" target="_blank">Click here</a>'})};
-
-            return api.notifications.browse({context: {internal: true}}).then(function then(results) {
-                if (!_.some(results.notifications, {message: notification.message})) {
-                    return api.notifications.add({notifications: [notification]}, {context: {internal: true}});
-                }
-            });
-        }).finally(function noMatterWhat() {
+        updateCheck().finally(function noMatterWhat() {
```

Comment:
general comment:
i would like to avoid connecting routes (in this case the `GET /ghost` route) with logic like checking if a new ghost version is available. in best case the user waits for one more database query (which is reading the settings) and in worst case the user waits for a database query, a request to our update service and an update of the notifications.

how i thought it would work:
1. a unit sends a long polling request to update service or we simply use a timer and wait x (everything in background)
2. store a message if new version available
3. `GET /notifications` from ember